### ISSUE TITLE
"caffe.LayerParameter" has no field named "multi_continuous_meanfield_param"

### ISSUE THREAD

**Issue by JamesDeng88** [danxuhk/ContinuousCRF-CNN#8]
The full error track:
WARNING: Logging before InitGoogleLogging() is written to STDERR
W0525 15:19:52.490731  2820 _caffe.cpp:139] DEPRECATION WARNING - deprecated use of Python interface
W0525 15:19:52.490770  2820 _caffe.cpp:140] Use this instead (with the named "weights" parameter):
W0525 15:19:52.490775  2820 _caffe.cpp:142] Net('/home/jdeng/w3_crf/ContinuousCRF-CNN/data/ResNet50_NYUDV2_deploy.prototxt', 1, weights='/home/jdeng/w3_crf/ContinuousCRF-CNN/data/ResNet50_NYUDV2.caffemodel')
[libprotobuf ERROR /thirdparty/protobuf/protobuf-3.2.0/src/google/protobuf/text_format.cc:299] Error parsing text-format caffe.NetParameter: 2410:36: Message type "caffe.LayerParameter" has no field named "multi_continuous_meanfield_param".
F0525 15:19:52.520838  2820 upgrade_proto.cpp:90] Check failed: ReadProtoFromTextFile(param_file, param) Failed to parse NetParameter file: /home/jdeng/w3_crf/ContinuousCRF-CNN/data/ResNet50_NYUDV2_deploy.prototxt
*** Check failure stack trace: ***

 I am getting this error, although I am using your caffe. Am I doing some mistakes?

**Comment by danxuhk** [comment]
If you check caffe.proto, the multi_continuous_meanfield_param is defined there. If the installation of the provided caffe version is successful, the error should not be there ...

**Comment by JamesDeng88** [comment]
I think you updated your github page recently. I will try to recompile everything with the latest code then let you know the result. Thank you for your quick relpy. 👍

**Comment by srikar8** [comment]
Cuda compilation tools, release 9.1, V9.1.85
ubuntu - 16.04


I am trying your model in other project, I am getting this error while trying it.

layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: MultiContinuousMeanfield (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, WindowData)
*** Check failure stack trace: ***

**Comment by srikar8** [comment]
solved.
thanks

**Comment by ZhenYangWuu** [comment]
you mean that  if i want to run the test.py i should build caffe again?
